### PR TITLE
Remove recursive submodules

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Remove some of recursive submodules. See #13378

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
The notorious usability of git: https://gist.github.com/myusuf3/7f645819ded92bda6677

File: base/daemon/SentryWriter.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ void setExtras()
     sentry_set_extra("version_patch", sentry_value_new_int32(VERSION_PATCH));
 }
 
-void sentry_logger(sentry_level_t level, const char * message, va_list args)
+void sentry_logger(sentry_level_e level, const char * message, va_list args, void *)
```
Comment (alesapin) [inline base/daemon/SentryWriter.cpp:49]: Changes in upstream?

**Review by alesapin — APPROVED** [review]
Just wait for tests, almost nothing to review.

File: base/daemon/SentryWriter.cpp (hunk 1 above)
Comment (alexey-milovidov) [reply base/daemon/SentryWriter.cpp:49]: Yes, I have updated it just in case.

**Comment by alesapin** [comment]
Failures are not relevant to changes.

**Pull request merged**